spot-vaults/contracts/WethWamplManager.sol

```diff
@@ -241,44 +245,52 @@ contract WethWamplManager {
 
     /// @notice Executes vault rebalance.
     function rebalance() public {
-        uint256 deviation = computeDeviationFactor();
-        uint256 activeLiqPerc = computeActiveLiqPerc(deviation);
+        (uint256 deviation, bool deviaitonValid) = computeDeviationFactor();
+        uint256 activeLiqPerc = deviaitonValid
```

Comment:
Maybe move this line below to the "execute rebalance" sections, since it's not used in the "update limit range" section

Reply:
Restructured with the newer version